src/Exceptions/Interfaces/ExceptionInterface.php

```diff
@@ -52,6 +52,12 @@
         'title' => 'Unprocessable entity',
     ];
 
+    // Billing/payment related errors 11000 - 11999
+    const PAYMENT_REQUIRED = [
+        'code'  => '11000',
+        'title' => 'Payment Required',
```

Comment:
Is this an error where payment is required? Or where billing details are missing?